app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt

```diff
@@ -67,17 +69,23 @@ class TopicInfoFragmentTest {
   }
 
   @Test
-  @Ignore("Landscape not properly supported") // TODO(#56): Reenable once landscape is supported.
   fun testTopicInfoFragment_loadFragment_configurationChange_checkTopicName_isCorrect() {
     launchTopicActivityIntent(TEST_TOPIC_ID).use { scenario ->
       scenario.onActivity {  activity ->
         activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
       }
-
       onView(withId(R.id.topic_name_text_view)).check(matches(withText(containsString(TOPIC_NAME))))
     }
   }
 
+  @Test
+  fun testTopicInfoFragment_loadFragment_configurationLandscape_isCorrect() {
+    launchTopicActivityIntent(TEST_TOPIC_ID).use {
+      onView(ViewMatchers.isRoot()).perform(OrientationChangeAction.orientationLandscape())
```

Comment:
You can directly import it: 
``onView(ViewMatchers.isRoot()).perform(orientationLandscape())``

Reply:
Once this is done, check there are no unused imports.